app/javascript/components/public_health/PatientsTable.js

```diff
@@ -93,6 +98,33 @@ class PatientsTable extends React.Component {
     // select tab and fetch patients
     this.handleTabSelect(tab);
 
+    // Select page if it exists in local storage
+    let page = localStorage.getItem(`${this.props.workflow}Page`);
```

Comment:
I think the page number should also be reset in between different queries. As of now, if you switch between tabs, the page number still stays the same and I would think that the users would probably want to see the first page of results when switching to a different filtered view?

Reply:
Agreed, I made this a bit smarter.